### DISCUSSION TITLE
How can I get the svn ice to work?

### DISCUSSION THREAD

**Discussion by carneloot** [z-shell/zi#299] [Q&A] [answered]
I'm trying to install the oh-my-zsh plugin for git-flow aliases and completions, but the svn ice is not working.

I'm simply trying to do
```sh
zi ice svn
zi snippet OMZP::git-flow
```

But it does not work. I keep getting the error:

```sh
Snippet: OMZP::git-flow
Downloading: `OMZP::git-flow` (with Subversion)…
svn: E170013: Unable to connect to a repository at URL 'https://github.com/ohmyzsh/ohmyzsh/trunk/plugins/git-flow'
svn: E160013: '/ohmyzsh/ohmyzsh/trunk/plugins/git-flow' path not found
```

How can I get this to work?

**Comment by ss-o** [comment, marked as answer]
Hi @carneloot :wave: 

The answer is you can't get SVN to work with GitHub, and the reasoning is described here: https://github.blog/2023-01-20-sunsetting-subversion-support/

We are considering the possible alternatives and will update the [wiki](https://wiki.zshell.dev) ASAP when a reasonable solution is found.

**Comment by carneloot** [comment]
Hey, thanks for the info! I couldn't find anything online and was getting worried. Now it all makes sense.
